source/_patterns/04-pages/03-teaser.mustache

```diff
@@ -42,9 +42,9 @@
 
     <div class="teaser teaser-50 teaser-offer">
       <a href="#" class="teaser-box" role="button">
-        {{> molecules-teaser-images }}
+        {{> molecules-top-offer-image }}
```

Comment:
... and here not. Correct?

Reply:
solved via call

Reply:
What was the answer during the call?

Reply:
It gets a default value, if it's not handed over :)

Reply:
```
{{> molecules-top-offer-image(imgSrc: '../../images/3-2-image-placeholder.png') }}
```
Will define an individual imgSrc

If it is not defined:
```
{{> molecules-top-offer-image }}
```
Then there will be a default. And in this case, the default is defined in data.json.